npm install error

```

[02:49:42] Finished 'check.versions' after 17 ms
npm ERR! Darwin 15.4.0
npm ERR! argv "/Users/white/.nvm/versions/node/v4.4.4/bin/node" "/Users/white/.nvm/versions/node/v4.4.4/bin/npm" "install"
npm ERR! node v4.4.4
npm ERR! npm  v2.15.1
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package rxjs@5.0.0-beta.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer angular2@2.0.0-beta.15 wants rxjs@5.0.0-beta.2
npm ERR! peerinvalid Peer @angular/core@2.0.0-rc.1 wants rxjs@5.0.0-beta.6

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/white/ng2-bootstrap-sbadmin/npm-debug.log


```

I already lost my faith about Nodejs and Angular2

I have the same problem.
@unnijeevan if you delete the caret in front of "ng2-bootstrap": "1.0.13", the UNMET PEER DEPENDENCY disappears. There are actually no errors while install. However, during start the following errors appear:

`[11:55:20] Starting 'build.js.dev'...
src/client/app/components/header/header.ts(4,9): error TS2305: Module '"/Users/n0daft/Documents/Entwicklung/Test/ng2-bootstrap-sbadmin-master/node_modules/ng2-bootstrap/components/dropdown"' has no exported member 'DropdownDirective'.
src/client/app/components/header/header.ts(4,28): error TS2305: Module '"/Users/n0daft/Documents/Entwicklung/Test/ng2-bootstrap-sbadmin-master/node_modules/ng2-bootstrap/components/dropdown"' has no exported member 'DropdownMenuDirective'.
src/client/app/components/header/header.ts(4,51): error TS2305: Module '"/Users/n0daft/Documents/Entwicklung/Test/ng2-bootstrap-sbadmin-master/node_modules/ng2-bootstrap/components/dropdown"' has no exported member 'DropdownToggleDirective'.
`

If you however use Angular version "2.0.0-rc.1" it will not install since the version is not found: 
`No compatible version found: angular2@2.0.0-rc.1`

--- a/package.json
+++ b/package.json
@@ -114,7 +114,7 @@
     "es6-shim": "0.35.0",
     "font-awesome": "^4.6.1",
     "moment": "^2.12.0",
-    "ng2-bootstrap": "^1.0.13",
+    "ng2-bootstrap": "1.0.13",
     "reflect-metadata": "0.1.2",
     "rxjs": "5.0.0-beta.2",
     "systemjs": "~0.19.25",

